lib/Chat/Parser/Changelog.php

```diff
@@ -28,13 +28,19 @@ public function handle(Event $event): void {
 			return;
 		}
 
-		if ($chatMessage->getActorType() !== Attendee::ACTOR_GUESTS ||
-			$chatMessage->getActorId() !== Attendee::ACTOR_ID_CHANGELOG) {
+		if ($chatMessage->getActorType() !== Attendee::ACTOR_GUESTS) {
 			return;
 		}
 
-		$l = $chatMessage->getL10n();
-		$chatMessage->setActor(Attendee::ACTOR_BOTS, Attendee::ACTOR_ID_CHANGELOG, $l->t('Talk updates ✅'));
-		$event->stopPropagation();
+		if ($chatMessage->getActorId() === Attendee::ACTOR_ID_CHANGELOG) {
+			$l = $chatMessage->getL10n();
+			$chatMessage->setActor(Attendee::ACTOR_BOTS, Attendee::ACTOR_ID_CHANGELOG, $l->t('Talk updates ✅'));
+			$event->stopPropagation();
+		}
+
+		if ($chatMessage->getActorId() === Attendee::ACTOR_ID_SAMPLE) {
+			$l = $chatMessage->getL10n();
+			$chatMessage->setActor(Attendee::ACTOR_BOTS, Attendee::ACTOR_ID_SAMPLE, $l->t('Nextcloud'));
```

Comment:
Just checking, it's correct that for `ACTOR_ID_SAMPLE` we don't need a `$event->stopPropagation` as we do for changelog?

Reply:
Also, I don't think we should translate `Nextcloud` ? What about theming/branding?

Reply:
> What about theming/branding?

Yeah I was wondering the same, we do use "Talk Updates" already, so if we use "Talk" here it should work branding-wise

> Just checking, it's correct that for ACTOR_ID_SAMPLE we don't need a $event->stopPropagation as we do for changelog?

We need to not have it here, as opposed to changelog, the samples can do mentions, shares, widgets, etc.

Reply:
The other "branding" solution would be to simply disable the example content or populate your own, but doesn't feel too good to push for disabling too heavily, so I would tend to "Talk"

Reply:
Switched the name to `Name` from `index.php/settings/admin/theming`